src/data/skill-desc.csv

```diff
@@ -1,123 +1,123 @@
-"阪神レース場が少し苦手になる","<size=22>Become slightly worse at Hanshin Racetrack</size>"
+"阪神レース場が少し得意になる","<size=22>Become a little better at Hanshin Racetrack</size>"
+"阪神レース場が少し苦手になる","<size=22>Become a little worse at Hanshin Racetrack</size>"
 "京都レース場が得意になる","<size=22>Become better at Kyoto Racetrack</size>"
-"京都レース場が少し得意になる","<size=22>Become slightly better at Kyoto Racetrack</size>"
-"京都レース場が少し苦手になる","<size=22>Become slightly worse at Kyoto Racetrack</size>"
+"京都レース場が少し得意になる","<size=22>Become a little better at Kyoto Racetrack</size>"
+"京都レース場が少し苦手になる","<size=22>Become a little worse at Kyoto Racetrack</size>"
 "中京レース場が得意になる","<size=22>Become better at Chukyo Racetrack</size>"
-"中京レース場が少し得意になる","<size=22>Become slightly better at Chukyo Racetrack</size>"
-"中京レース場が少し苦手になる","<size=22>Become slightly worse at Chukyo Racetrack</size>"
+"中京レース場が少し得意になる","<size=22>Become a little better at Chukyo Racetrack</size>"
+"中京レース場が少し苦手になる","<size=22>Become a little worse at Chukyo Racetrack</size>"
 "札幌レース場が得意になる","<size=22>Become better at Sapporo Racetrack</size>"
-"札幌レース場が少し得意になる","<size=22>Become slightly better at Sapporo Racetrack</size>"
-"札幌レース場が少し苦手になる","<size=22>Become slightly worse at Sapporo Racetrack</size>"
+"札幌レース場が少し得意になる","<size=22>Become a little better at Sapporo Racetrack</size>"
+"札幌レース場が少し苦手になる","<size=22>Become a little worse at Sapporo Racetrack</size>"
 "函館レース場が得意になる","<size=22>Become better at Hakodate Racetrack</size>"
-"函館レース場が少し得意になる","<size=22>Become slightly better at Hakodate Racetrack</size>"
-"函館レース場が少し苦手になる","<size=22>Become slightly worse at Hakodate Racetrack</size>"
+"函館レース場が少し得意になる","<size=22>Become a little better at Hakodate Racetrack</size>"
+"函館レース場が少し苦手になる","<size=22>Become a little worse at Hakodate Racetrack</size>"
 "福島レース場が得意になる","<size=22>Become better at Fukushima Racetrack</size>"
-"福島レース場が少し得意になる","<size=22>Become slightly better at Fukushima Racetrack</size>"
-"福島レース場が少し苦手になる","<size=22>Become slightly worse at Fukushima Racetrack</size>"
+"福島レース場が少し得意になる","<size=22>Become a little better at Fukushima Racetrack</size>"
+"福島レース場が少し苦手になる","<size=22>Become a little worse at Fukushima Racetrack</size>"
 "新潟レース場が得意になる","<size=22>Become better at Niigata Racetrack</size>"
-"新潟レース場が少し得意になる","<size=22>Become slightly better at Niigata Racetrack</size>"
-"新潟レース場が少し苦手になる","<size=22>Become slightly worse at Niigata Racetrack</size>"
+"新潟レース場が少し得意になる","<size=22>Become a little better at Niigata Racetrack</size>"
+"新潟レース場が少し苦手になる","<size=22>Become a little worse at Niigata Racetrack</size>"
 "小倉レース場が得意になる","<size=22>Become better at Ogura Racetrack</size>"
-"小倉レース場が少し得意になる","<size=22>Become slightly better at Ogura Racetrack</size>"
-"小倉レース場が少し苦手になる","<size=22>Become slightly worse at Ogura Racetrack</size>"
+"小倉レース場が少し得意になる","<size=22>Become a little better at Ogura Racetrack</size>"
+"小倉レース場が少し苦手になる","<size=22>Become a little worse at Ogura Racetrack</size>"
```

Comment:
What do you think about rewording all of the `a little` to `a bit` since they're shorter?